package.json

```diff
@@ -133,6 +146,7 @@
     "start": "node run",
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "test:cov": "npm test -- —coverage"
+    "test:cov": "npm test -- --coverage",
+    "test:ci": "node --harmony_proxies node_modules/.bin/jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
```

Comment:
IMO the submission to Coveralls should happen in `after_success` inside Travis, not here. I'm pretty sure this will lead to errors if executed locally b/c we won't have the Coveralls token configured.